dipy/workflows/reconst.py

```diff
@@ -941,27 +943,30 @@ def run(self, input_files, bvalues_files, bvectors_files, split_b_D=400,
                                               b0_threshold=0)
 
             if not save_metrics:
-                save_metrics = ['S0_est', 'f_est', 'D_star_est', 'D_est']
+                save_metrics = ['S0_predicted', 'perfusion_fraction', 'D_star',
+                                'D']
 
-            S0_est = ivimfit.model_params[..., 0]
```

Comment:
@arokem sorry for the delay, but this is what you wanted me to do right? Let me know if I misunderstood!